### DISCUSSION TITLE
Exporting JSON about repositories

### DISCUSSION THREAD

**Discussion by astrochun** [PyGithub/PyGithub#1946] [Q&A]
I have a code that access `/users/{user}/repos` endpoint to retrieve a list of repositories for a user and generate a `pandas` DataFrame of the metadata (e.g., URLs, etc).

```
endpoint = f"https://api.github.com/users/{user}/repos"
params = {
    'per_page': 100
}

response = requests.get(endpoint, params=params)
repository_list = json.loads(response.content)
repository_df = pd.DataFrame.from_dict(repository_list)
```

As part of extended capability, I want to now use `PyGitHub`.  With `GitHub().get_user(user).git_repos()` this provides a paginated list of `Repository` with the metadata as instance variables. Is there not an easy way to get the normal JSON response from this endpoint using PyGithub?

**Comment by s-t-e-v-e-n-k** [comment]
We wrap every response in some sort of object so you can query properties and call methods of them. There is a `raw_data` property if you want the JSON itself.

**Reply by astrochun** [reply]
Thanks. After posting, I did find that the Repository does have `raw_data` and `_rawData` properties; however, this wasn't easily found at first attempt in the RTDs. See: https://pygithub.readthedocs.io/en/latest/github_objects/Repository.html#repository

**Comment by s-t-e-v-e-n-k** [comment]
Yes, we'd prefer they remain they aren't immediately discoverable. :-)

**Reply by astrochun** [reply]
I see. While I understand, it should be noted that there is a gap in explanation/information for how to go between standard GitHub API responses toward `PyGitHub` objects.
